lms/services/blackboard_api/_schemas.py

```diff
@@ -20,6 +20,7 @@ class Meta:
         id = fields.Str(required=True)
         name = fields.Str(required=True)
         modified = fields.Str(required=True)
+        type = fields.Str(required=True)
```

Comment:
The view needs the `type` field from the Blackboard API so that it can tell files from folders. This field gets sent through to the frontend too.